<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>551</width>
    <height>396</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="patient_records_list_widget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>551</width>
      <height>321</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="open_record_button">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>330</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Open Record</string>
    </property>
   </widget>
   <widget class="QPushButton" name="add_new_record_button">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>330</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Add New Record</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>551</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSearch_Patient">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionShow_Patients"/>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuSearch_Patient"/>
  </widget>
  <action name="actionShow_Patients">
   <property name="text">
    <string>Show Patients</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
